build/style_test.go

```diff
@@ -1015,15 +1015,15 @@ func TestStyle(t *testing.T) {
 				// which results in the unused warning:
 				//
 				// sql/parser/yaccpar:362:3: this value of sqlDollar is never used (SA4006)
-				&lint.GlobIgnore{Pattern: "github.com/cockroachdb/cockroach/pkg/sql/parser/yaccpar", Checks: []string{"SA4006"}},
+				&lint.GlobIgnore{Pattern: "github.com/cockroachdb/cockroach/pkg/sql/parser/sql.go", Checks: []string{"SA4006"}},
```

Comment:
Why this change? Do the comments need an update? Ditto below.

Reply:
This is just reverting a previous change that was made necessary by an upstairs bug (see the comment being removed from Gopkg.toml)

Reply:
Er, I misspoke - there's no comment referring to the upstream bug, but here's where it got fixed: https://github.com/dominikh/go-tools/commit/80dfe5569e5c609daca40943b2385b4c6cc33b48